lib/practice.md

```diff
@@ -3,58 +3,61 @@
 - What package did you install for enumeration?
 
 ```md
-<!-- your answer here -->
+lodash
 ```
 
 ## Research: Popular NPM Packages
 
 Package 1:
 ```md
-<!-- your answer here -->
+express
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+routing
 ```
 1.  How can I use this package in my own code? What steps are involved?
 ```md
-<!-- your answer here -->
+you can use it with routing methods instead of ajax, I think...
```

Comment:
It can be used to create routes, but Ajax is still required on the front end if Express.js is used as a back end API, or if it's used to communicate with a third party API.